static.config.js

```diff
@@ -386,29 +386,6 @@ const makeDepartmentPages = async (client, langCode) => {
     if (!infoPage.department) continue;
 
     infoPage.type = 'info';
```

Comment:
I don't remember what we're using this for/if we're using it anywhere. Might be able to drop this entire block